packages/webapp/src/webview/ui/components/App.scss

```diff
@@ -19,11 +25,16 @@ h1 {
         z-index: 10;
         background-color: var(--vscode-editor-background);
         &__sub {
-            display: flex;
-            align-items: center;
+            // display: flex;
```

Comment:
Hi @sanat-thukral, is there a reason why this styles are commented out and still in the code? I'll remove them before merging, they are still in the history in case you need them.